Assets/MRTK/Providers/XRSDK/XRSDKDeviceManager.cs

```diff
@@ -43,8 +43,12 @@ public virtual bool CheckCapability(MixedRealityCapability capability)
         protected static readonly Dictionary<InputDevice, GenericXRSDKController> ActiveControllers = new Dictionary<InputDevice, GenericXRSDKController>();
 
         private readonly List<InputDevice> inputDevices = new List<InputDevice>();
+        private readonly List<InputDevice> inputDevicesSubset = new List<InputDevice>();
         private readonly List<InputDevice> lastInputDevices = new List<InputDevice>();
 
+        // Need to make it so we get devices that are either controllers or handtracking, the old commented code only got devices that met both requirements
+        protected static readonly InputDeviceCharacteristics[] DesiredCharacteristics = new InputDeviceCharacteristics[]
+            {InputDeviceCharacteristics.Controller, InputDeviceCharacteristics.HandTracking, InputDeviceCharacteristics.Controller | InputDeviceCharacteristics.HandTracking};
```

Comment:
```suggestion
            {InputDeviceCharacteristics.Controller, InputDeviceCharacteristics.HandTracking};
```

Reply:
I think all devices that would match this case will match either of the first two cases as well

Reply:
The GetDevicesWithCharacteristics function is an exact match, so we need to do it this way to get devices that have handtracking, controller or both properties (I think, I'm not sure what the previous case was we could test this one)

Reply:
I don't think it's an exact match, because all XR SDK input sources contain several (HL2 hands have ~6 or 7 I believe) marked characteristics.

This API checks the characteristic flags for all flags you pass in though, so these three will look for:

1. All sources that are marked as a controller
1. All sources that are marked as hand tracking
1. All sources that are marked as both a controller and hand tracking

I can't think of a way for group 3 to contain sources that weren't already matched in the first two groups.

Reply:
ohhh I see, will adjust